common/persistence/src/main/resources/db/migration/V19__alterTableScanInstanceAtTestRegistration.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE scan_instances_at_test_registration
+  ALTER COLUMN seconds_since_last_scan TYPE INTEGER,
+  ALTER COLUMN typical_attenuation TYPE INTEGER,
+  ALTER COLUMN minimum_attenuation TYPE INTEGER;
+
+ALTER SEQUENCE scan_instances_at_test_registration_id_seq AS bigint;
+
+ALTER TABLE scan_instances_at_test_registration ALTER COLUMN id TYPE bigint;
```

Comment:
@ioangut hopefully this works with the large amount of data... next time I recommend to first alter the table and then afterwards the sequence... I don't know how the system will behave, when the sequence already changed, but the column not